static/css/components/cbox.less

```diff
@@ -50,10 +48,12 @@
     cursor: pointer;
   }
   &Content {
+    width: 660px !important;
```

Comment:
This looks like it would break some modal's (ex: change a book cover, view book cover, etc.). Why are we fixing the width/height?